assets/css/map/controls/_street_view_switch.scss

```diff
@@ -0,0 +1,72 @@
+// BEM-PREFIX: LC = Leaflet Control
+.lc-street-view-switch {
+  // #region Position
+  position: absolute;
+  // Leaflet uses px for positioning of their controls so we use px here to match
+  top: 10px;
+  right: 55px;
+  // #endregion
+
+  // #region Look
+  user-select: none;
+
+  color: $color-eggplant-700;
+  background-color: $white;
+  // #endregion
+
+  // #region Layout
+  display: flex;
+  align-items: center;
+  gap: 4px;
+
+  font-size: 1rem;
+  line-height: 1.5rem;
+
+  padding: 8px;
+
+  border: 1px solid $color-gray-400;
+  border-radius: 7px;
+  // #endregion
+
+  // #region Sub-Components
+  &__label {
+    padding-right: 0.5rem;
+  }
+
+  &__label-icon svg {
+    fill: currentColor;
+
+    height: 1em;
+    height: calc(1em - 2px);
+    min-width: 16px;
+  }
+
+  &__input {
+    // Shrinkwrap the wrapper component to the input component so that flexbox
+    // can align this element properly
+    min-height: unset;
+    height: min-content;
+
+    // Remove layout offsets
+    padding-left: 0;
+    margin-bottom: 0;
+
+    // Remove Negative Margins so shrinkwrap works
+    .form-check-input {
+      margin-left: 0;
+    }
+  }
+  // #endregion
+
+  // #region Media Queries
+  @media screen and (max-width: 800px) {
```

Comment:
I couldn't figure out a good way to factor this out into a variable. Suggestions welcome!

Reply:
I think ideally we'd do something like a map in SCSS of the various breakpoint sizes to replace the outdated variables like `$mobile-max-width`, although I don't have strong opinions on things like where it would live.

Reply:
Maybe we do something along these lines? https://medium.com/free-code-camp/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862

Reply:
Opened a ticket for this to be addressed later 👍🏻 https://app.asana.com/0/1152340551558956/1204009922904919/f